src/test/java/com/alkemy/ong/bigtest/BigTest.java

```diff
@@ -261,4 +262,8 @@ protected String convert(Object requestObject) throws JsonProcessingException {
     return objectMapper.writeValueAsString(requestObject);
   }
 
+  protected Object convert(String jsonString,Class<?> type) throws JsonProcessingException {
+    return objectMapper.readValue(jsonString,type);
+  }
+
```

Comment:
NO REQUIRED: comment in src/test/java/com/alkemy/ong/bigtest/category/CreateCategoryIntegrationTest.java